com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitGUI.cs

```diff
@@ -72,6 +85,11 @@ public struct LitProperties
             public MaterialProperty highlights;
             public MaterialProperty reflections;
 
+            public MaterialProperty clearCoat;  // Enable/Disable dummy property
```

Comment:
we should add API docs (XmlDocs) for all new public API. We will address the missing docs at some point.

Reply:
What about the EnableKeyword(_CLEARCOAT) etc.?